packages/wallets/src/features/cfd/flows/CTrader/AvailableCTraderAccountsList/AvailableCTraderAccountsList.tsx

```diff
@@ -51,10 +58,9 @@ const AvailableCTraderAccountsList: React.FC = () => {
 
     return (
         <TradingAccountCard
+            disabled={isCFDAccountCreationLoading || isCFDAccountCreationSuccess}
             leading={<div className='wallets-available-ctrader__icon'>{PlatformDetails.ctrader.icon}</div>}
-            onClick={() => {
-                onSubmit();
-            }}
+            onClick={onSubmit}
```

Comment:
what's the difference here?

Reply:
Nothing. Just directly referencing the onSubmit function 😅